app/models/adjective.rb

```diff
@@ -0,0 +1,4 @@
+class Adjective < ApplicationRecord
+  has_many :descriptions
+  has_many :users, through: :descriptions, source: :describee
```

Comment:
@Dpalazzari @wlffann What is the purpose of this has many relationship? Do we have any need to go from an adjective to all users that have been described with that adjective right now?